### DISCUSSION TITLE
Easy Update

### DISCUSSION THREAD

**Discussion by chris-der-kleine** [Pryaxis/TShock#2078] [General]
I just want to thank you for the stupid easy upgrade path!

Typically updating takes ages and half of the time it doesn't even work!  
To update my docker image, I just throw the new files in my image folder, recompile and it just works😍
This takes less than 5 minutes and minimizes downtime. AWESOME!

Everything should be this simple. Thank you for that❤

**Comment by hakusaro** [comment]
Aww, I'm glad you liked the process!
GitHub <notifications@github.com> wrote:
“I just want to thank you for the stupid easy upgrade path!

Typically updating takes ages and half of the time it doesn't even work!

 To update my docker image, I just throw the new files in my image folder, recompile and it just works😍
 This takes less than 5 minutes and minimizes downtime. AWESOME!

Everything should be this simple. Thank you for that❤”
